tests/utils/random.ts

```diff
@@ -1,18 +1,63 @@
-import { pseudoRandomRange } from "../../exports/base";
-
-/**
- * Creates a function which can be called infinite times, each call instance will yield pseudo random number in [0, 1];
- * @param initialSeed The seed.
- * @returns The generator.
- */
-export function createPseudoRandomGenerator01(initialSeed: number) {
-    const g = (function *(): Generator<number, never, never> {
-        for (let seed = initialSeed; ; ++seed) {
-            yield pseudoRandomRange(seed, 0, 1);
+export class PseudoRandomGenerator {
+    constructor(initialSeed: number) {
+        this._seed = initialSeed;
+    }
+
+    get defaultOrder() { return this._defaultOrder }
+    set defaultOrder(value) { this._defaultOrder = value; }
+
+    /** Generates number in [min, max]. */
+    public range(min: number, max: number) {
+        if (min > max) {
+            throw new Error(`min should be less than max.`);
```

Comment:
As the comment says, should do it like
```typescript
if (min >= max) 
```

Reply:
If min equals max is valid, then should modify the description.

Reply:
https://github.com/cocos/cocos-engine/pull/15547/commits/03459c1ff8fe9e4650a02dd1dce624c2d1be1658